wiki/Manual.md

```diff
@@ -1581,6 +1563,31 @@ As such, `cancelOrder()` can throw an `OrderNotFound` exception in these cases:
 - canceling an already-closed order
 - canceling an already-canceled order
 
+
+## Trades / Transactions / Fills / Executions
+
+```
+- this part of the unified API is currenty a work in progress
+- there may be some issues and missing implementations here and there
+- contributions, pull requests and feedback appreciated
+```
+
+A trade is a result of order execution. Note, that orders and trades have 1-n relationship: execution of 1 order may result in several trades.
+
+
+##### Recent Trades
```

Comment:
The docs are propagated to various places, one of them is http://ccxt.readthedocs.io/ that requires a special format, generated with Sphinx. It is overly critical to the structure of the document, so the header levels should be properly nested, it does not like gaps like between ## and #### header (should be a ### in between).

Reply:
Also, if we change the sections of the Manual, we also have to change this outline: https://github.com/ccxt/ccxt/wiki (generated from the wiki/README.md index file).

Reply:
Let me know if you want me to fix it, I'm happy to help.